micro-ui/web/micro-ui-internals/packages/modules/hrms/src/pages/Response.js

```
@@ -1,115 +1,173 @@
-        data={mutation?.data|| successData}
-        action={state.action}
-        isSuccess={!successData ? mutation?.isSuccess : true}
-        isLoading={(mutation.isIdle && !mutationHappened) || mutation?.isLoading}
-        isEmployee={props.parentRoute.includes("employee")}
+    <>
+      <PanelCard
+        type={isResponseSuccess ? "success" : "error"}
+        message={t(state?.message || "SUCCESS")}
+        response={`${state?.showID ? state?.id : ""}`}
+        footerChildren={[]}
+        children={state?.showChildren ? children : null}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)